Cargo.toml

```diff
@@ -14,8 +14,67 @@ members = [
   'pallets/circuit-breaker',
   'pallets/omnipool-liquidity-mining',
   'scraper',
+  'pallets/faucet',
+  'traits',
+  'pallets/relaychain-info',
+  'pallets/price-oracle',
+  'pallets/route-executor',
+  'pallets/transaction-multi-payment',
+  'pallets/asset-registry',
+  'runtime/adapters',
+  'pallets/collator-rewards',
+  'pallets/transaction-pause',
+  'pallets/ema-oracle',
+  'pallets/liquidity-mining',
+  'pallets/currencies',
+  'pallets/stableswap',
+  'utils/test-utils',
+  'pallets/dynamic-fees',
+  'pallets/duster',
+  'pallets/nft',
+  'pallets/ema-oracle',
```

Comment:
```suggestion
```
for some reason in ended up twice in the list?

Reply:
its my favorite